contributors/guide/owners.md

```diff
@@ -118,17 +120,17 @@ emeritus_approvers:
 #### Cleanup
 
 In addition to the Emeritus process above, from time to time, it is necessary
-to prune inactive members from OWNERS files. A core principle in maintaining a
-healthy community is encouraging active participation. OWNERS have a higher
-activity requirement, as they directly impact the ability of others to contribute.
-If an owner should become inactive, they will be removed but may be added back
-should they become active again.
+to prune inactive folks from OWNERS files. A core principle in maintaining a
+healthy community is encouraging active participation. folks listed in OWNERS files
+have a higher activity requirement, as they directly impact the ability of others
+to contribute. If anyone listed in OWNERS files should become inactive, they will
+be removed but may be added back should they become active again.
```

Comment:
Are they removed, or added to emeritus_approvers?  Your code PR https://github.com/kubernetes/kubernetes/pull/107293/files seemed to move them to emeritus, but you seem to imply here they are completely removed.  Which is it?

Reply:
reviewers are completely removed, approvers are bumped to `emeritus_approvers`.

Reply:
I'll add this in a minute

Reply:
Updated PR